policies/templates/gcp_dataproc_location_v1.yaml

```diff
@@ -0,0 +1,103 @@
+# Copyright 2019 Google LLC
```

Comment:
Is this change intended to be in this PR?

Reply:
it was not originally, but merging both policies will fix my branches on my fork for future PRs. Sorry for the confusion.

Reply:
I removed it from this PR